tests/foreman/api/test_activationkey.py

```diff
@@ -126,6 +130,20 @@ def test_negative_create_3(self):
                         unlimited_content_hosts=True,
                     ).create()
 
+    def test_negative_create_4(self):
```

Comment:
In the mean time, there is no need for a complete module refactor, just starting this approach from now on will freeze the effort needed to rename old tests.